lua/pac3/core/client/parts/interpolated_multibone.lua

```diff
@@ -0,0 +1,230 @@
+local BUILDER, PART = pac.PartTemplate("base_drawable")
+
+PART.ClassName = "interpolated_multibone"
+PART.Group = 'advanced'
+PART.Icon = 'icon16/table_multiple.png'
+PART.is_model_part = false
+
+PART.ManualDraw = true
+PART.HandleModifiersManually = false
+
+BUILDER:StartStorableVars()
+	:SetPropertyGroup("test")
+		:GetSet("Preview", false)
+	:SetPropertyGroup("Interpolation")
+		:GetSet("LerpValue",0)
+		:GetSet("Power",1)
+		:GetSet("InterpolatePosition", true)
+		:GetSet("InterpolateAngles", true)
+	:SetPropertyGroup("Nodes")
```

Comment:
Likewise, these are parts, not nodes

Reply:
Would be nice to have a single part list property, but not sure at the top of my head how that should work.

Reply:
We can rename these with editor_friendly fields no problem. But yes, in my opinion those are nodes. Nodes are points, as in a graph. That's what a base_movable part can do if we take its position. But that's just linguistic talk. It doesn't change how the code works.

As for making it dynamic, I know it would've been nice. But I decided not to mess around very much in the deeper parts of the pac code like dynamic properties.